var/spack/repos/builtin/packages/possvm/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Possvm(Package):
+    """Possvm (Phylogenetic Ortholog Sorting with Species oVerlap and MCL)
+    is a python tool to analyse pre-computed gene trees and identify pairs
+    and clusters of orthologous genes. It takes advantage of the species
+    overlap algorithm implemented in the ETE toolkit to parse the phylogeny
+    and identify orthologous gene pairs, and MCL clustering for orthogroup
+    identification."""
+
+    homepage = "https://github.com/xgrau/possvm-orthology"
+    git = "https://github.com/xgrau/possvm-orthology.git"
+
+    license("GPL-3.0-only", checked_by="A-N-Other")
+
+    # version number is taken from -v/--version argparse info in possvm.py
+    version("1.2", commit="8e97f27806095b9368836e7630188f9b8d473b55")
```

Comment:
Personally I would choose https://github.com/xgrau/possvm-orthology/commit/3158757423edafc29aa29bf3ae0cc63a93a56df9 as version 1.2 since that's the first commit where that version appears. Could also just use the date of the commit as the version since they have no official releases.